examples/overlay/README.md

```diff
@@ -0,0 +1,6 @@
+# Overlay example
+
+The OVERLAY operation accepts (a message with) xxxxxxx and returns (a message with) results xxxxx.
```

Comment:
@finnagin @ShaopengLiu1 What is intended behind the `xxxxxxx`?

Reply:
I think this may have been a placeholder to edit once we hammered out the details.

TODO: @finnagin to flesh this out to be similar to the readme in `fill_and_filter`.